sycl/test/extensions/usm/usm_alloc_utility.cpp

```diff
@@ -0,0 +1,82 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t1.out
+// RUN: %RUN_ON_HOST %t1.out
+
+//==------ usm_alloc_utility.cpp - USM malloc and aligned_alloc test -------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <CL/sycl.hpp>
+
+#include <cassert>
+
+using namespace cl::sycl;
+
+constexpr int N = 8;
+
+int main() {
+  queue q;
+  auto dev = q.get_device();
+  auto ctxt = q.get_context();
+  int *array;
+
+  if (dev.get_info<info::device::usm_host_allocations>()) {
+    array = (int *)malloc_host(N * sizeof(int), q);
+    assert((get_pointer_type(array, ctxt) == usm::alloc::host) &&
+           "Allcoation pointer should be host type");
```

Comment:
Typo.

Reply:
That took me longer to find than I'd like to admit. Good eye!